exercises/practice/bob/.approaches/answer-array/snippet.txt

```diff
@@ -0,0 +1,14 @@
+    // uses System.Linq;
+    private static readonly string[] answers = {"Whatever.", "Sure.", "Whoa, chill out!", "Calm down, I know what I'm doing!"};
+    
+    public static string Response(string statement)
+    {
+        var input = statement.TrimEnd();
+        if (input == "") return "Fine. Be that way!";
+        
+        var isShout = (input.Any(c => char.IsLetter(c)) && input.ToUpper() == input) ? 2: 0;
+        
+        var isQuestion = (input.EndsWith('?')) ? 1: 0;
+        
+        return answers[isShout + isQuestion];
+    }
```

Comment:
I think there's an 8-line limit to the snippet.txt file. Or am I imagining that?

@ErikSchierboom can you confirm?

Reply:
There is. We don't yet enforce that (configlet doesn't yet check the approaches files)